src/views/poi_panel.dot

```diff
@@ -8,12 +8,18 @@
        {{= this.panel.renderPartial(this.headerPartial) }}
        {{= this.minimalHourPanel.render() }}
       </div>
+      <div class="poi_panel__content__card__action__container">
         <button class="poi_panel__content__card__action" {{= click(this.showDetail, this) }}>
           <span class="icon-chevrons-right"></span>
           {{= _('SEE MORE', 'poi panel') }}
         </button>
+        <button class="poi_panel__content__card__action poi_panel__content__card__action__direction" {{= click(this.openDirection, this) }}>
+          <span class="icon-chevrons-right"></span>
+          {{= _('DIRECTION', 'poi panel') }}
```

Comment:
* wrong icon
* directions button should be the first one
* `'DIRECTION'` :arrow_right: `'DIRECTIONS'`